test/e2e/specs/customer_admin.go

```diff
@@ -233,15 +234,17 @@ var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fa
 	It("should sync AAD admin group", func() {
 		syncSecret, err := sanity.Checker.Client.Admin.CoreV1.Secrets("openshift-infra").Get("aad-group-sync-config", metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred())
-		id := "44e69b4e-2e70-42df-bb97-3a890730d7b0"
-		testUser := "testuserdisabled"
-		gid := syncSecret.Data["customerAdminGroupId"]
-		if strings.EqualFold(string(gid), id) {
-			// test the users
+
+		var provider api.AADIdentityProvider
+		err = yaml.Unmarshal(syncSecret.Data["aad-group-sync.yaml"], &provider)
```

Comment:
Just to make sure I'm understanding this right, looking at e.g. https://github.com/openshift/openshift-azure/blob/28ad9dd9b76bfc016186e01c679ed601b27ac08f/pkg/sync/v3/data/Secret/openshift-infra/aad-group-sync-config.yaml#L3-L7

Was the `gid` previously null and hence the test assertions below weren't invoking? Because I don't see a customerAdminGroupId defined that it could have pulled the value from.

Reply:
it is set by the CI system (from a secret) exported here: https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml#L292
so it should be set in e2e jobs (this should be set to the AAD group called "CI test" which has one user called "testuserdisabled").

Reply:
So the CustomerAdminGroupID is always set, but the retrieved user.Mail was nil. i don't think this e2e test change is strictly needed, as the bug would cause the sync process to fail and it would already fail this test.

Reply:
ok, I might not get this, but this fix is because configMap structure changed?

Reply:
I thought I replied to this, but I guess I didn't.
@ehashman you are right.
@asalkeld the fix here is that the test was not ever running